charts/nextcloud/values.yaml

```diff
@@ -580,7 +579,37 @@ cronjob:
 
   # The command the cronjob container executes.
   command:
-  - /cron.sh
+    - /cron.sh
+
+  # Use a CronJob instead of crond sidecar container
+  # crond does not work when not running as root user
+  # Note: requires `persistence.enabled=true`
+  useCronJobInsteadOfCrond: false
+  labels: {}
+  annotations: {}
+  affinity: {}
+    # Often RWO volumes are used.
+    # Depending on your provider two pods on the same node can still write to the same volume.
+    # Following config ensures that the cronjob pod is scheduled on the same node as the nextcloud pod.
+    # podAffinity:
+    #   requiredDuringSchedulingIgnoredDuringExecution:
+    #     - labelSelector:
+    #         matchExpressions:
+    #           - key: app.kubernetes.io/name
+    #             operator: In
+    #             values:
+    #               - nextcloud
+    #           - key: app.kubernetes.io/component
+    #             operator: In
+    #             values:
+    #               - app
+    #       topologyKey: kubernetes.io/hostname
+  commandCronJob:
+    - php
+    - -f
+    - /var/www/html/cron.php
+    - --
+    - --verbose
```

Comment:
I feel like someone is going to get confused between the two commands, however im not sure if there is anything we can do to prevent confusion here.

Reply:
I get the point. But I feel like `external` could be confusing as well. As per "outside of this chart" etc.
I can't come up with better name than `cronjob.cronjob` maybe `cronjob.kubernetesCronJob`. But that feels weird/long as well.

Implemented the suggestion.

Reply:
Hey apologies, I will have to get back to you on testing this one, I have hit rate limits and I cannot properly template and test the helm chart as I cannot pull half of the chart's dependencies.

Partly due to this change just announced and trying to prepare for it: https://github.com/bitnami/containers/issues/83267 which also affects the dependencies of this chart. :/ 6 hour wait for this rate limit is killer!